README.plugin.md

```diff
@@ -7,6 +7,18 @@
 See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin)
 for how to install Foreman plugins
 
+## Migrating your plugin:
```

Comment:
The wiki's the main source of detailed maintainer docs, it should probably be changed first. I'd suggest leaving it out of here entirely, I don't think it's important in the plugin README - the only bit that may be useful is uninstallation, but this probably doesn't implement the complete process?